src/redux/modules/resellers.js

```diff
@@ -0,0 +1,68 @@
+import { resellerService } from 'api';
+
+// ------------------------------------
+// Constants
+// ------------------------------------
+
+const GET_RESELLERS_PENDING = 'backoffice:get-resellers:pending';
+const GET_RESELLERS_SUCCESS = 'backoffice:get-resellers:success';
+const GET_RESELLERS_ERROR = 'backoffice:get-resellers:error';
+
+// ------------------------------------
+// Actions
+// ------------------------------------
+
+export function getResellers() {
+  return dispatch => dispatch({
+
+    types: [
+      GET_RESELLERS_PENDING,
+      GET_RESELLERS_SUCCESS,
+      GET_RESELLERS_ERROR
+    ],
+    payload: {
+      promise: resellerService.get()
+        .then(response => response)
+    }
+
+  });
+}
+
+// ------------------------------------
+// Reducers
+// ------------------------------------
+
+type Action = {
+  type: string;
+  payload: any;
+}
```

Comment:
Separate types to their own header
// ------------------------------------
// Type definitions
// ------------------------------------